proto/blueprint.proto

```diff
@@ -545,6 +545,26 @@ message Card {
    * Indictate where to display the headline.
    */
   optional HeadlinePosition headline_position = 43;
+
+  /**
+   * Define card sizes
+   */
+  enum CardSize {
+    CARD_SIZE_UNSPECIFIED = 0;
+    CARD_SIZE_XX_SMALL = 1;
```

Comment:
Shall we remove this for now - given it's only necessary for sublinks?

Reply:
Yeah agree.